Typescript compilation fails (Cannot find type definition file for 'ws' / 'request')

## Expected behaviour

When importing this lib in a typescript project, we should be able to compile using the libs types. 

## Actual behaviour

```
node_modules/twitch-js/index.d.ts:9:23 - error TS2688: Cannot find type definition file for 'ws'.

9 /// <reference types="ws" />
                        ~~

node_modules/twitch-js/index.d.ts:10:23 - error TS2688: Cannot find type definition file for 'request'.

10 /// <reference types="request" />
                         ~~~~~~~

node_modules/twitch-js/index.d.ts:13:56 - error TS2307: Cannot find module 'request'.

13 import { RequestCallback, Options as ApiOptions } from 'request';
                                                          ~~~~~~~~~
```

## Steps to reproduce

From a new folder:

- `npm init`
- `npm install -D typescript twitch-js`
- create a file `index.ts` containing:

```typescript
import { Client } from "twitch-js";

export function test() {
  const c = new Client();
}
```
- `.\node_modules\.bin\tsc .\index.ts`

It outputs the error above.

Maybe this is a case where `@types/request` and `@types/ws` should be moved from `devDependencies` to `dependencies`? like described here: https://stackoverflow.com/a/46011417/1245409

## Your environment

* Operating system(s): Windows 10
* TwitchJS version: 1.2.16
* NodeJS version: 10.13.0

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "twitch-js",
-  "version": "1.2.17-pre.0",
+  "version": "1.2.17",
   "description": "Javascript library for the Twitch Messaging Interface.",
   "license": "MIT",
   "author": "Schmoopiie",

